privacyidea/lib/smsprovider/SmppSMSProvider.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+#
+#    E-mail: yurlov.alexandr@gmail.com
+#
+#    2017-12-22 Alexander Yurlov <yurlov.alexandr@gmail.com>
+#               Add new SMPP SMS provider
+#
+#
+#    Copyright (C) LinOTP: 2010 - 2014 LSE Leading Security Experts GmbH
```

Comment:
Did you actually copy it from https://github.com/LinOTP/LinOTP/blob/master/linotpd/src/linotp/provider/smsprovider/SMPPSMSProvider.py or did you create it on your on, based on the parent class?
Then you might remove this copyright note!

Reply:
I've copied this header from near files in master branch. I'll remove it